etl/helpers.py

```diff
@@ -50,3 +54,38 @@ def _get_github_branches(org: str, repo: str) -> List[Any]:
         raise Exception("reached single page limit, should paginate request")
 
     return branches
+
+
+class Naming:
+    def __init__(self, file: str):
+        self.f = Path(file)
+
+    @property
+    def channel(self) -> str:
+        return self.f.parent.parent.parent.name
+
+    @property
+    def namespace(self) -> str:
+        return self.f.parent.parent.name
+
+    @property
+    def version(self) -> str:
+        return self.f.parent.name
+
+    @property
+    def short_name(self) -> str:
+        return self.f.with_suffix("").name
+
+    @property
+    def country_mapping_path(self) -> Path:
+        return self.f.parent / (self.short_name + ".country_mapping.json")
+
+    @property
+    def metadata_path(self) -> Path:
+        return self.f.parent / (self.short_name + ".meta.yml")
+
+    @property
+    def meadow_dataset(self) -> catalog.Dataset:
+        return catalog.Dataset(
+            paths.DATA_DIR / f"meadow/{self.namespace}/{self.version}/{self.short_name}"
+        )
```

Comment:
This is a reasonable assumption that will cover most cases, but in principle the meadow version of a dataset can be different from the garden version.

Reply:
For sure, these are all just recommendations that might help if you're a newcomer (or don't want to think about it), but you're free to use full path.